docs/topics/proc_exposing-application-metrics-using-prometheus-with-vertx.adoc

```diff
@@ -159,9 +159,9 @@ public class CustomVertxApp extends AbstractVerticle {
 
     vertx.setPeriodic(1000, l -> {
       timer.record(() -> {
-      
```

Comment:
Is this line just indentation fix? :)

Reply:
It seems to me like dangling white space.

Reply:
Hmm. Ok. Excuse me for being dense but is the issue on line 162?

Reply:
@huntley Basically just why you did it.

As a side note, it is considered a semi-bad practice to include unrelated changes like removing dangling whitespace in commits. Is it what happened? (You don't have to edit this PR it if so, just make sure you do white space removal in a separate PR, ideally for the entire doc set at the same time.)

Reply:
Ahh. The spacing change was inadvertent. What is the easiest way to revert that change?

Reply:
@huntley Don't worry about it, it's too much work and it's not really "forbidden." Unless you did that deliberately, what I suspect is that your editor's set up to remove dangling white space automatically. If that's the case, I'd only ask you to find that setting and turn it off.